src/cryptonote_core/service_node_list.cpp

```diff
@@ -1185,6 +1181,15 @@ namespace service_nodes
     return true;
   }
 
+  static uint64_t quorum_rng_seed(crypto::hash const &hash, quorum_type type)
+  {
+    uint64_t result = 0;
+    std::memcpy(&result, hash.data, sizeof(result));
+    boost::endian::little_to_native_inplace(result);
+    result += static_cast<uint64_t>(type);
+    return result;
+  }
```

Comment:
mt19937 has significantly more than 64-bit of seed values; seed_seq is how you feed it more.  It will pull all the values out of the seed_seq to seed itself, so if we feed it 9 values, it'll use those 9x32 bits for the seed.

Reply:
Alright, I've changed `shuffle_portable` to take an rng, and we construct the rng with seed_seq on HF16 onwards.